rs/bitcoin/kyt/tests/tests.rs

```diff
@@ -110,7 +195,7 @@ fn test_get_inputs() {
     // There wll be two outcalls because the canister will first fetch the above
     // given txid, and then fetch the vout[0] from the returned transaction body.
 
-    let canister_http_requests = tick_until_next_request(&env);
+    let canister_http_requests = tick_until_next_request(env);
```

Comment:
here for a lack of a better place: I think we should also test scenarios, where 1 of the HTTPs outcalls give us an error (e.g. a 500 because the provider is down or 429 due to rate-limit, consensus error, etc)

Reply:
Thanks for the suggestion! Added a test for 500